### PR TITLE
Updated chocolatey package to 1.1 

### PR DESCRIPTION
**Pull request by yetanotherchris**

### CODE REVIEW

**Comment by dbolkensteyn** [comment]
btw @yetanotherchris you didn't have to close the previous PR - they get updated as you add more commit

**Comment by dbolkensteyn** [comment]
thanks

File: msbuild-sonarqube-runner/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -1,5 +1,8 @@
+$url = "https://github.com/SonarSource-VisualStudio/sonar-msbuild-runner/releases/download/1.1/MSBuild.SonarQube.Runner-1.1.zip"
+$checksum = "D9302247D6AAFAB2B76256BFC96FC8F74468B87A"
+
 ﻿Install-ChocolateyZipPackage 'msbuild-sonarqube-runner' `
```
Comment (dbolkensteyn) [inline msbuild-sonarqube-runner/tools/chocolateyinstall.ps1:4]: @yetanotherchris seems like you left the UTF-8 BOM character on this line, whereas it should be moved to the new first line - I'm taking care to fix that

**Comment by dbolkensteyn** [comment]
Merged in https://github.com/SonarSource/chocolatey-packages/commit/5714c1bb0638597212fef10bcc2005c9f434bb90

**Comment by yetanotherchris** [comment]
Ah ok that's worth knowing

File: msbuild-sonarqube-runner/tools/chocolateyinstall.ps1 [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
-  -url 'https://github.com/SonarSource/sonar-msbuild-runner/releases/download/1.0/MSBuild.SonarQube.Runner-1.0.zip' `
+  -url '$url' `
```
Comment (dbolkensteyn) [inline msbuild-sonarqube-runner/tools/chocolateyinstall.ps1:5]: btw double quotes are required to have the variable expanded

**Comment by dbolkensteyn** [comment]
fyi the new version is waiting to for approval from Chocolatey:

https://chocolatey.org/packages/msbuild-sonarqube-runner/1.1.0.20160205

**Comment by yetanotherchris** [comment]
@dbolkensteyn thanks, did you fix the single quotes issue? I learnt something new about PS :+1:  I'm a double quoter usually

**Comment by dbolkensteyn** [comment]
i did, the updated package is live now on chocolatey: it has been approved

**Pull request closed without merging**